### PR TITLE
Fix GH-2639: Mention empty needle in string functions

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/php/doc-en/issues/2639

### CODE REVIEW

File: reference/strings/functions/stripos.xml [hunk 1]
```diff
@@ -40,10 +40,8 @@
      <term><parameter>needle</parameter></term>
      <listitem>
       <para>
-       Note that the <parameter>needle</parameter> may be a string of one or
-       more characters.
+       The string to search for.
       </para>
-      &strings.parameter.needle.non-string;
```
Comment (Girgias) [inline reference/strings/functions/stripos.xml:46]: Why did this get removed?

File: language-snippets.ent [hunk 2]
```diff
@@ -3947,6 +3947,14 @@ local: {
...
 '>
 
+<!ENTITY strings.changelog.needle-empty '<row xmlns="http://docbook.org/ns/docbook">
+ <entry>8.0.0</entry>
+ <entry>
+  <parameter>needle</parameter> now accepts an empty string.
+ </entry>
+</row>'>
+
+
```
Comment (Girgias) [inline language-snippets.ent:3957]: Nit double new line

**Review by Girgias — COMMENTED** [review]
One question but LGTM

File: reference/strings/functions/stripos.xml (hunk 1 above)
Comment (alexandre-daubois) [reply reference/strings/functions/stripos.xml:46]: It shouldn't, mishandling on my part. Restored :slightly_smiling_face:

**Pull request merged**